{"timestamp":"2025-09-03T07:08:42.900672","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-03T07:08:42.901610","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/test_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-03T07:08:42.942920Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-03T07:08:42.943051Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-03T07:08:42.943228Z","level":"info","event":"Current task name:casse","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-03T07:08:42.943306Z","level":"info","event":"Dag name:exemple_simple","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-03T07:08:42.945508","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"ZeroDivisionError","exc_value":"division by zero","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/test_dag.py","lineno":7,"name":"probleme"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-03T07:08:42.972999Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-03T07:08:42.973168Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-03T07:08:42.973237Z","level":"info","event":"Task:<Task(PythonOperator): casse>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-03T07:08:42.973324Z","level":"info","event":"Failure caused by division by zero","chan":"stdout","logger":"task"}
